src/styles/generic.css

```diff
@@ -0,0 +1,95 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&family=Open+Sans:wght@400;500;600;700;800&family=Poppins:wght@400;500;600;700;800&family=Roboto:wght@300;400;500;700;900&display=swap');
+
+:root {
+  --bg-dark: #2b2727;
+  --black: #000;
+  --text-color: #fff;
+  --primary-txt: #f00;
+}
+
+html {
+  box-sizing: border-box;
+  font-size: 100%;
+}
+
+*,
+*::after,
+*::before {
+  box-sizing: inherit;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: 'Roboto', sans-serif;
+  font-size: 14px;
+  background-color: var(--bg-dark);
+  color: var(--text-color);
+}
+
+main {
+  min-height: 100vh;
+}
+
+header {
+  display: flex;
+  margin: 0 auto;
+  align-items: center;
+  padding-inline: 13px;
+  margin-bottom: 12px;
+  background-color: var(--black);
+}
+
+header ul {
+  list-style: none;
+  display: flex;
+  column-gap: 16px;
+  margin-left: auto;
+}
+
+header ul a {
+  text-decoration: none;
+  color: var(--text-color);
+  transition: all 0.2s ease-in-out;
+}
+
+header ul a:hover {
+  color: var(--primary-txt);
+}
+
+header ul li.active {
+  text-decoration: underline;
+}
+
+footer {
+  width: 90%;
```

Comment:
- [ ] I would suggest making the footer width 100% or 95% so that it will looks great and adding some margin as well.